python/py_helpers.test.py

```diff
@@ -298,6 +298,17 @@ def test_whitespace_equivalence(self):
             Node(str_with_whitespace).is_equivalent(str_with_different_whitespace)
         )
 
+    def test_string_equivalence(self):
+        self.assertTrue(Node("'True'").is_equivalent('"""True"""'))
```

Comment:
Is there a case where a developer might use `Node(code).is_equivalent...`? If so, this feels like more of a gotcha.

Reply:
I'm not sure exactly how people will use this, but, `ast.parse('"""thing"""')` and `ast.parse("'thing'")` are the same up to column offsets. That makes it tricky to distinguish between the two.

Maybe this is confusing, but I suspect not _too_ confusing. After all `'thing' == "thing" == """thing"""` and `is_equivalent` has the same relationship.

Reply:
Ah. I had actually forgotten Python had `"""`